### PR TITLE
[website]: improve rpc navbar layout

### PR DESCRIPTION
**Pull request by jmcook1186**
moves page links in "interacting with Geth" section of LH navbar to a more intuitive layout.

### CODE REVIEW

File: src/data/documentation-links.yaml [hunk 1]
```diff
@@ -46,36 +46,38 @@
...
+            to: /docs/interacting-with-geth/rpc/ns-eth
+          - id: les
+            to: /docs/interacting-with-geth/rpc/ns-les
+          - id: miner
+            to: /docs/interacting-with-geth/rpc/ns-miner
+          - id: net
+            to: /docs/interacting-with-geth/rpc/ns-net
+          - id: Personal (deprecation notes)
+            to: /docs/interacting-with-geth/rpc/ns-personal-deprecation
+          - id: Personal (deprecated)
```
Comment (fjl) [inline src/data/documentation-links.yaml:67]: Why are there two pages related to `personal`? I think we could have the reason for deprecation at the top of the reference docs page.

Comment (jmcook1186) [reply src/data/documentation-links.yaml:67]: yea we could - that just creates a very long page. The deprecation notes include mapping out the deprecated methods to their replacements.

Your call - if you prefer a very long single page then no problem

Comment (fjl) [reply src/data/documentation-links.yaml:67]: I think it'd be better to have only a single page. I don't think it's a problem to have a long page, it's more important to have a 'canonical' page for the personal namespace. Long term, we will remove this page anyway.

**Comment by fjl** [comment]
I guess we should add a redirect also?

**Comment by jmcook1186** [comment]
ah, good point, thanks. Added the redirect.

**Pull request merged**